dashboard/app/helpers/district_dropdown_constants.rb

```diff
@@ -0,0 +1,8 @@
+module DistrictDropdownConstants
```

Comment:
Since this is not only consumed in the view, I think it makes more sense in the model instead of a (view) helper. Then also you won't need to require it all over the place as it will be auto-included by rails.

Yesterday we were talking about potentially adding validations to this common class too. If you're still thinking about doing that, perhaps a more general name (e.g. just DistrictDropdown) that doesn't limit us to just constants?